app/src/main/java/org/wikipedia/readinglist/database/ReadingListDbHelper.java

```diff
@@ -171,12 +182,27 @@ public void addPageToList(@NonNull ReadingList list, @NonNull PageTitle title, b
         }
     }
 
+    public void movePageToList(long sourceReadingListId, @NonNull ReadingList list, @NonNull PageTitle title, boolean queueForSync) {
```

Comment:
Can you just use `movePagesToListIfNotExist` instead?

Reply:
Yes!